src/app/component/dashboard/customers/customers.component.html

```diff
@@ -124,7 +124,7 @@
 <modal #customer modalClass="modal-lg" (keydown.esc)="confirmClose()">
   <modal-header>
     <h4>{{ language.createNewCustomer }}</h4>
-    <button type="button" class="close" data-dismiss="modal" [attr.aria-label]="cancelButtonLabel || 'Close'" (click)="confirmClose()">
+    <button type="button" class="custom-close" data-dismiss="modal" [attr.aria-label]="cancelButtonLabel || 'Close'" (click)="confirmClose()">
```

Comment:
Can you pull changes first from "MergeBranch" first, because I deleted [attr.aria-label]="cancelButtonLabel || 'Close'"
We have a problem with build.